docs/advisors/checks/mongodb-chunk-imbalance.md

```diff
@@ -0,0 +1,37 @@
+# MongoDB sharding - chunk imbalance across shards
+
+## Description
+This check warns if the chunks are imbalanced across shards.
+
+## Resolution
+
+In a sharded cluster, chunk imbalances can occur when the data is not evenly distributed among the shards. This can lead to some shards having more chunks than others. This can cause performance issues and slower query times.
```

Comment:
```suggestion
In a sharded cluster, chunk imbalances can occur when the data is not evenly distributed among the shards. This can lead to some shards having more chunks than others, which can cause performance issues and slower query times.
```